associations/urls.py

```diff
@@ -26,9 +25,10 @@
 
 associations_router = routers.NestedSimpleRouter(router, r'associations', lookup='association')
 associations_router.register(r'events', EventViewSet)
-#router.register(r'associations/(?P<association_id>[a-zA-Z0-9-_]+)/events', EventViewSet)
+associations_router.register(r'news', NewsAssociationViewSet)
 
 urlpatterns = [
+    path('news/', NewsSubscriptionsViewSet.as_view()),
```

Comment:
Are the news routes registered twice here ? (line 28 ans 31)

Reply:
No, one is for the `NewsAssociationViewSet`, the other for `NewsSubscriptionsViewSet` (which I renamed `NewsSubscriptionsView` by the way).